src/pages/sitemap-docs.xml.tsx

```diff
@@ -0,0 +1,31 @@
+import { GetServerSidePropsContext } from 'next'
+
+export default function SiteMap() {
+	return null
+}
+
+const siteUrl = process.env.SITE_URL || 'https://developer.hashicorp.com'
+
+export async function getServerSideProps({ res }: GetServerSidePropsContext) {
```

Comment:
@nayres Are you thinking that the ideal state will be that `next-sitemap` handles the generation at build time? I saw your note about the issue you ran into, but just confirming that your intention is that this is a "first step."

FWIW, I'm sort of undecided on if this runtime solution, or a build-time solution, is better 🤔

Reply:
Figured it out!